helpers/deploymentConfig.ts

```diff
@@ -9,39 +9,21 @@ interface BridgeConfig {
 const SEPOLIA_MULTISIG = "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb";
 
 export const preconfiguredAddresses = {
-  hardhat: {
-    VTreasury: "account:deployer",
-  },
   bsctestnet: {
-    VTreasury: "0x8b293600C50D6fbdc6Ed4251cc75ECe29880276f",
-    NormalTimelock: "0xce10739590001705F7FF231611ba4A48B2820327",
-    FastTrackTimelock: "0x3CFf21b7AF8390fE68799D58727d3b4C25a83cb6",
-    CriticalTimelock: "0x23B893a7C45a5Eb8c8C062b9F32d0D2e43eD286D",
-    AccessControlManager: "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA",
     XVS: "0xB9e0E753630434d7863528cc73CB7AC638a7c8ff",
-    ResilientOracle: "0x3cD69251D04A28d887Ac14cbe2E14c52F3D57823",
     LzEndpoint: "0x6Fcb97553D41516Cb228ac03FdC8B9a0a9df04A1",
     LzVirtualChainId: "10102",
   },
   bscmainnet: {
-    VTreasury: "0xF322942f644A996A617BD29c16bd7d231d9F35E9",
-    NormalTimelock: "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396",
-    FastTrackTimelock: "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02",
-    CriticalTimelock: "0x213c446ec11e45b15a6E29C1C1b402B8897f606d",
-    AccessControlManager: "0x4788629ABc6cFCA10F9f969efdEAa1cF70c23555",
     XVS: "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63",
-    ResilientOracle: "0x6592b5DE802159F3E74B2486b091D11a8256ab8A",
     LzEndpoint: "0x6592b5DE802159F3E74B2486b091D11a8256ab8A",
     LzVirtualChainId: "102",
   },
   sepolia: {
-    VTreasury: "0xFc43c055B9be2Ec3BEe6f8C291Af862d764016a0",
     NormalTimelock: SEPOLIA_MULTISIG,
     FastTrackTimelock: SEPOLIA_MULTISIG,
     CriticalTimelock: SEPOLIA_MULTISIG,
-    AccessControlManager: "0x799700ea540f002134C371fB955e2392FD94DbCD",
-    XVS: "0xD657eB80daA42c334B1c70Cb274E83E4163A3dDc",
-    ResilientOracle: "0x1B85FAbE5c0846662F5FB0E3598fC48eF587e9f0",
+    ResilientOracle: "0x9005091f2E0b20bEf6AaF2bD7F21dfd45DA8Af07",
```

Comment:
It should be available in the npm package of the oracles, since 1.8.0-dev.2

Reply:
[9d3a335](https://github.com/VenusProtocol/token-bridge/pull/2/commits/9d3a335283ddfbf9ba26a3f1fe81a008a114e21e)